### PR TITLE
Design feedback on update notification

### PR DESCRIPTION
**Pull request by shakyShane**
**Asana Task/Github Issue:** https://app.asana.com/0/1201141132935289/1208684961069930/f
## Description

- [x] Design feedback applied

## Testing Steps

- visit: https://deploy-preview-1190--content-scope-scripts.netlify.app/build/pages/new-tab/?update-notification=populated
- Ensure the `dismiss` icon is always present

## Checklist

*Please tick all that apply:*

- [ ] I have tested this change locally
- [ ] I have tested this change locally in all supported browsers
- [ ] This change will be visible to users
- [ ] I have added automated tests that cover this change
- [ ] I have ensured the change is gated by config
- [ ] This change was covered by a ship review
- [ ] This change was covered by a tech design
- [ ] Any dependent config has been merged

### CODE REVIEW

**Comment by shakyShane** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1190?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#1150** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1150?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1190** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1190?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1176** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1176?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @shakyShane and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: special-pages/pages/new-tab/app/components/Layout.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { h } from 'preact'
+
+export function Centered ({ children }) {
+    return (
+        <div class="layout-centered">
```
Comment (mgurgel) [inline special-pages/pages/new-tab/app/components/Layout.js:5]: Is the plan to eventually adopt <Centered> across the board and move `.layout-centered` out of the global namespace?

File: special-pages/pages/new-tab/app/update-notification/UpdateNotification.module.css [hunk 2]
```diff
@@ -56,17 +53,9 @@
...
-    transition: all .3s;
-    position: absolute;
-    top: -2px;
-    right: var(--sp-2);
-}
-
 .detailsContent {
     padding-inline: var(--sp-2);
-    padding-block: var(--sp-4);
+    margin-top: var(--sp-2);
```
Comment (mgurgel) [inline special-pages/pages/new-tab/app/update-notification/UpdateNotification.module.css:58]: Why not margin-block-start for consistency?

**Review by mgurgel — APPROVED** [review]
Looks good. Left a couple of questions, none of which blockers.

File: special-pages/pages/new-tab/app/components/Layout.js (hunk 1 above)
Comment (shakyShane) [reply special-pages/pages/new-tab/app/components/Layout.js:5]: I wanted to do that, but then I realised that I'm using it elsewhere as an additional class... not sure what's best right now, let's revisit when there's some pain :)

**Pull request merged**